index.js

```diff
@@ -31,6 +31,7 @@ function choo (opts) {
   start.router = router
   start.model = model
   start.start = start
+  start.stop = _store.stop
```

Comment:
sure, I've just been using like this in my fork because it was super simple and it works! Did you have any particular api/implementation in mind?